### PR TITLE
fix(linux): fix tab completion for command line tools

### LINKED ISSUE #5306: bug(linux): tab completion doesn't work for files with km-package-install
**Describe the bug**

`km-package-install -f <TAB>` should allow for tab completion for files. However it appears not to work.

---

**Keyman for Windows/macOS/Linux:**
 - OS: Ubuntu 20.04 
 - Keyman version: 14.0.276 
 - Target Application: terminal

### PR DESCRIPTION
**Pull request by ermshiperete**
Fixes #5306

### CODE REVIEW

File: linux/keyman-config/km-config [hunk 1]
```diff
@@ -16,7 +16,10 @@ if __name__ == '__main__':
     parser.add_argument('--version', action='version', version='%(prog)s version ' + versionstring)
     parser.add_argument('-v', '--verbose', action='store_true', help='verbose logging')
     parser.add_argument('-vv', '--veryverbose', action='store_true', help='very verbose logging')
-    parser.add_argument('-i', '--install', action='store', help='download and/or install .kmp package')
+    parser.add_argument('-i', '--install', action='store', help='download and/or install .kmp ' +
+                        'package. INSTALL can either be a downloaded .kmp file, a file:// URL ' +
+                        'pointing to a .kmp file, or a keyman:// URL, possibly with a ' +
+                        'bcp47=<language> specified.')
```
Comment (darcywong00) [inline linux/keyman-config/km-config:22]: Are the `-h --help` args already added?

From 
```
opts="-h --help -v --verbose -vv --veryverbose --version -i --install"
```

**Review by darcywong00 — APPROVED** [review]
lgtm

File: linux/keyman-config/km-config (hunk 1 above)
Comment (ermshiperete) [reply linux/keyman-config/km-config:22]: `-h` and `--help` are built-in arguments that get added implicitly.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.75-alpha](https://keyman.com/downloads/releases/alpha/15.0.75)

**Pull request merged**